packages/react-native/src/Authenticator/Defaults/SignIn/SignIn.tsx

```diff
@@ -50,6 +53,18 @@ const SignIn = ({
   const signInText = getSignInText();
   const signUpText = getSignUpTabText();
 
+  const body = (
+    <>
+      {socialProviders ? (
+        <FederatedProviderButtons
+          route="signIn"
+          socialProviders={socialProviders}
+          toFederatedSignIn={toFederatedSignIn}
+        />
+      ) : null}
+    </>
+  );
```

Comment:
Think this can just be:
```
const body = socialProviders ? (
    <FederatedProviderButtons
      route="signIn"
      socialProviders={socialProviders}
      toFederatedSignIn={toFederatedSignIn}
    />
) : null;
```